packages/playwright-test/src/reporters/html.ts

```diff
@@ -429,7 +435,7 @@ class HtmlBuilder {
         return {
           name: a.name,
           contentType: a.contentType,
-          path: 'data/' + sha1,
+          path: this._attachmentsBaseURL ? this._attachmentsBaseURL + sha1 : 'data/' + sha1,
```

Comment:
path.join to allow no trailing slash.

Reply:
Tried to but it seems that path.join doesn't work well with URLs and it removes one slash from `https://` and gives `https:/`. Should I leave it as it is or sanitize path in some other way?